cmd/embedded-cluster/shell.go

```diff
@@ -99,7 +99,12 @@ var shellCommand = &cli.Command{
 
 		// if /etc/bash_completion is present enable kubectl auto completion.
 		if _, err := os.Stat("/etc/bash_completion"); err == nil {
-			config = fmt.Sprintf("source <(kubectl completion %s)\n", filepath.Base(shpath))
+			config = fmt.Sprintf("source <(k0s completion %s)\n", filepath.Base(shpath))
+			_, _ = shellpty.WriteString(config)
+			_, _ = io.CopyN(io.Discard, shellpty, int64(len(config)+1))
+
+			comppath := defaults.PathToEmbeddedClusterBinary("kubectl_completion_bash.sh")
```

Comment:
This only supports bash for now but I dont believe we supported anything else since we check for `/etc/bash_completion`